starter-project-backend-g33/src/app/comments/routes.ts

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express'
+import commentController from './controller'
+
+const router = Router()
+
+router.get('/', commentController.getAllComments)
+router.get('/:id', commentController.getComment)
+router.put('/:id', commentController.updateComment)
+router.post('/create', commentController.createComment)
+router.delete('/remove/:id', commentController.deleteComment)
```

Comment:
can you change the route here from
```/remove/:id``` -> ```/:id``` 

This is to keep it consistent with other routes in the other routers. The HTTP verbs should be enough to differentiate routes